hummingbot/connector/test_support/mock_paper_exchange.pyx

```diff
@@ -65,7 +65,8 @@ cdef class MockPaperExchange(PaperTradeExchange):
         return True
 
     def split_trading_pair(self, trading_pair: str) -> Tuple[str, str]:
-        return trading_pair.split("-")
+        base, quote = trading_pair.split("-")
```

Comment:
why this change is necessary?

Reply:
This is because of the stricter conformance to typing, `trading_pair.split("-")` returns a List, `split_trading_pair()` is supposed to return a `Tuple`